packages/table/table.scss

```diff
@@ -52,6 +52,7 @@ $highlight-color: $helhvit;
         }
 
         &--data-row {
+            -webkit-transition: background-color 150ms ease, box-shadow 150ms ease;
```

Comment:
This should be handled by autoprefixer?

Reply:
yeah, I think I ran autoprefixer on sass files while testing, and this slipped through